doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc

```diff
@@ -145,6 +145,133 @@ $ oc create -f cfme-restore-job.yaml
 
 After the restore job is complete, you can scale the pods back up.
 
+
+
+[[updating-pods]]
+=== Updating {product-title_short}
+
+To apply package updates to a {product-title_short} appliance running on OpenShift Container Platform, delete the pods and redeploy them using updated {product-title_short} container images. If you have modified the template, this requires additional steps in xref:modified-template-update[].
+
+ifdef::cfme[]
+
+See the https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html/release_notes/index#technical_notes[Technical Notes] in the _Release Notes_ for a list of updated {product-title_short} container images and errata.
+
+[IMPORTANT]
+====
+If you are updating a {product-title} 4.6.0 environment to version 4.6.1, follow the steps in xref:modified-template-update[]. When updating between these versions, the template must also be updated to reflect the updated ConfigMap mount path for `postgresql`. 
+====
+endif::cfme[]
+
+ifdef::miq[]
+[IMPORTANT]
+====
+If you are updating a {product-title} 5.9.0 environment to version 5.9.1, follow the steps in xref:modified-template-update[]. When updating between these versions, the template must also be updated to reflect the updated ConfigMap mount path for `postgresql`.
```

Comment:
4.6 and 5.9 are both downstream versions. I don't think this applies to upstream. I believe manageiq had the templates updated for Gaprindashvili-1.

Reply:
Ah ok. I've deleted this note entirely now as it doesn't seem necessary to mention it as the template update will resolve this automatically.